Source/GRChomboCore/ChomboParameters.hpp

```diff
@@ -347,6 +409,14 @@ class ChomboParameters
                         plot_interval <= 0 || plot_prefix != checkpoint_prefix,
                         "should be different to checkpoint_prefix");
 
+        check_parameter("output_path", output_path,
+                        GRParmParse::folder_exists(output_path),
+                        "should be a valid folder");
+        check_parameter("pout_path", pout_path,
+                        GRParmParse::folder_exists(pout_path),
+                        "should be a valid folder");
```

Comment:
Actually the folder was created on your laptop because even the `mkdir_recursive` for pout's created it, since it's recursive!!! But had you not used a pout/hdf5/.dat subpath, then it would not have been created. So what we mean by "must exist" is probably that we do not take direct responsibility in creating it ourselves.
I will remove the pout check -> indeed we create it ourselves, so it will exist if nothing goes wrong. The `output_path` check will give an error if the directory doesn't exist and you didn't specify any pout subpath (and hence `output_path` was not created by `mkdir_recursive`)